### ISSUE TITLE
Eating a mice cube is as bad as a monkey cube

### ISSUE THREAD

**Issue by Killette2** [d3athrow/vgstation13#23715]
#### Description of issue

Eat a mouse cube and die

#### Difference between expected and actual behavior

I expected to be like that one girl in those gross videos and eat a mouse
Then the mouse exploded in my chest, broke my organs, and I died

#### Steps to reproduce

Eat a mouse cube. I think they're only on roid atm

**Comment by Kharkov72** [comment]
I see no issue here

**Comment by Killette2** [comment]
It's a size thing. How would a tiny (malformed from being chewed) mouse inflate and do that much damage? It breaks muh immursions for it to break my ribs and crush my lungs and heart

**Comment by tuypo1** [comment]
I agree, they should still be deadly but maybe kill a little slower or something
